lookout/core/cmdline.py

```diff
@@ -75,7 +75,7 @@ def run_analyzers(args):
     data_service = DataService(data_request_address)
     log.info("Created %s", data_service)
     manager = AnalyzerManager(
-        analyzers=[importlib.import_module(a).analyzer_class for a in args.analyzer],
+        analyzers={a: importlib.import_module(a).analyzer_class for a in args.analyzer},
```

Comment:
Module names seem too hacky here. Besides, we can import the same modules by different paths and screw up the configuration. The better solution is to require `NAME` for every `Analyzer` - like we do in Modelforge models. That `NAME` should uniquely identify the class, can include all the qualification, etc.

Reply:
I pushed an edit implementing `name`. Should I also edit the cli so that it receives this name and not the module name we use currently? Pros: consistent, cons: longer to type

Reply:
The cli logic is simple and straightforward: it imports the module and takes the class. Nobody cares about the names except the developers who want to get the right config in their analyzer.